pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1856,7 +1856,14 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
           throw new Error('No container engine is set');
         }
         if (kubernetesVersion) {
-          await this.kubeBackend.install(config, kubernetesVersion, this.#adminAccess);
+          if (!config.kubernetes.version) {
+            const versionSetting = { kubernetes: { version: kubernetesVersion.version } };
+
+            await this.kubeBackend.install(merge(config, versionSetting), kubernetesVersion, this.#adminAccess);
+            mainEvents.emit('settings-write', versionSetting);
+          } else {
+            await this.kubeBackend.install(config, kubernetesVersion, this.#adminAccess);
```

Comment:
Is there any reason not to update the version anyway? It's possible that the actually found version will not match the requested version (maybe it's a different patch release)?